enhanced-resolve/lib/Storage.d.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference types="node" />
+import { CommonFileSystemMethod } from './common-types';
+
+declare class Storage {
+    duration: number;
+    running: {
+        [name: string]: ((this: null, ...args: any[]) => any)[]
+    };
+    data: {
+        [name: string]: any[]
+    };
+    levels: string[][];
+    count: number;
+    interval: NodeJS.Timer | null;
+    needTickCheck: boolean;
+    nextTick: number | null;
+    passive: boolean;
+
+    constructor(duration: number);
+
+    ensureTick(): void;
+
+    finished(name: string): void;
+
+    provide(name: string, provider: CommonFileSystemMethod, callback: (this: null, ...args: any[]) => any): any;
```

Comment:
It's just that it will be `null` even in sloppy mode, but typescript doesn't do sloppy mode anymore anyway :tada: